plane/src/drone/executor.rs

```diff
@@ -212,6 +212,9 @@ impl<R: Runtime> Executor<R> {
                     let Some(manager) = self.backends.get(backend_id) else {
                         tracing::warn!(backend_id = backend_id.as_value(), "Backend not found when handling terminate action (assumed terminated).");
 
+                        // Terminate will only return an error on a docker error, not if the backend is already terminated.
+                        self.runtime.terminate(backend_id, true).await?;
```

Comment:
Do we want to log anything based on whether this returned `Ok(true)` or `Ok(false)`?

Reply:
The `Ok(false)` case already logs a warning for the Docker runtime, I think that will be sufficient for now.